modules/architecture-machine-roles.adoc

```diff
@@ -58,28 +58,51 @@ master static Pods and etcd static Pods working on the same hosts.
 ====
 
 Services that fall under the Kubernetes category on the master include the
-API server, etcd, controller manager server, and HAProxy services.
+Kubernetes API server, etcd, Kubernetes controller manager, and HAProxy services.
 
 .Kubernetes services that run on the control plane
-
-[options="header"]
+[cols="1,2",options="header"]
 |===
 |Component |Description
-|API Server
+|Kubernetes API server
 |The Kubernetes API server validates and configures the data for Pods, Services,
 and replication controllers. It also provides a focal point for cluster’s shared
```

Comment:
An old sentence, but I guess this was supposed to be saying "for a cluster's shared state"? Though may be better to rewrite as "the shared state of a cluster", per IBM SG's guidance on avoiding inanimate possessives.